src/rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -262,6 +262,9 @@ if [[ -n $PAI_CODE_DIR ]]; then
   if [ $code_dir_size -gt 0 ] && [ $code_dir_size -le 200000000 ]; then
```

Comment:
I am confused about when to use single bracket and when to use double in the bash. Is there any convention in this project?

https://github.com/Microsoft/pai/blob/0f80208482eade027af88c61b4d58b78bf39b57a/src/rest-server/src/templates/yarnContainerScript.mustache#L256